src/arch/arch_default.h

```diff
@@ -39,14 +39,10 @@
 #define DEFAULT_INPUT_DRIVER_LIST "X11"
 #endif
 #define DEFAULT_MOVIE_DRIVER_LIST "FFMpeg,Null"
-// ALSA comes first, as the system may have OSS compat but we don't want to use
-// it if it's actually an ALSA wrapper.
-// Then try OSS before daemon drivers so we're going direct instead of
-// unwittingly starting a daemon.
-// JACK gives us an explicit option to NOT start a daemon, so try it third,
-// as PulseAudio will successfully Init() but not actually work if the
-// PulseAudio daemon has been suspended by/for jackd.
-#define DEFAULT_SOUND_DRIVER_LIST "ALSA-sw,OSS,JACK,Pulse,Null"
+// PulseAudio is the preferred Unix driver. It is the most accurate,
```

Comment:
I wouldn't say pulesaudio is the most accurate, especially since it's a layer on top of alsa. I would reword this as:

```cpp
// PulseAudio is the preferred Unix driver since it allows the gives non-exclusive access to the
// audio device, unlike ALSA.
// Use ALSA next because it is the lowest latency.
// Then try OSS before daemon drivers so we're going direct instead of
// unwittingly starting a daemon.
// JACK gives us an explicit option to NOT start a daemon, so try it third,
// as PulseAudio will successfully Init() but not actually work if the
// PulseAudio daemon has been suspended by/for jackd.
```